ONSdigital/sdc-responses-dashboard
# RAS-1003 Remove structlog.threadlocal

# What and why?
Update the logs to be thread safe. I have aligned them to our other services. Why we would punch out the logger name who knows, that has been removed

# How to test?
Deploy this PR and make sure the logs are coming out as expected you will need a CE with at least one respondent enrolled. http://localhost:8078/dashboard/

N.B I was getting back 404's from reporting (different repo) on mine for empty CEs or ones with just dummy rus (i.e 1111) that isn't connected to this PR, but the logs might have found an issue that wasn't shown before because of misconfigurations. A fix would need to be done in the reporting repo if there is a problem  


# Trello
